pkg/genericapiserver/resource_config.go

```diff
@@ -170,3 +171,15 @@ func (o *ResourceConfig) AnyResourcesForVersionEnabled(version unversioned.Group
 
 	return versionOverride.Enable
 }
+
+func (o *ResourceConfig) AnyResourcesForGroupEnabled(group string) bool {
```

Comment:
not the most complicated method, but we should have a test in resource_config_test.go